roles/micado_master/start/tasks/terraform-config.yml

```diff
@@ -14,4 +14,5 @@
     mode: '0644'
   loop:
     - { src: 'egi/configure', dest: '/var/lib/micado/toscasubmitter/system/configure'}
-    - { src: 'egi/pyGetScopedToken.py', dest: '/var/lib/micado/toscasubmitter/system/pyGetScopedToken.py'}
\ No newline at end of file
+    - { src: 'egi/pyGetScopedToken.py', dest: '/var/lib/micado/toscasubmitter/system/pyGetScopedToken.py'}
```

Comment:
Aren't these related to Terraform? Wouldn't they be better in `/var/lib/micado/terraform`?

Reply:
That is a good point.